File: python/problem6.py
#!/usr/bin/env python

def solve():
	sum_of_squares = reduce(lambda x,y: x + y, (i**2 for i in range(1,101)))
	sum = reduce(lambda x,y: x + y, (i for i in range(1,101)))
	square_of_sums = sum**2
	print "Problem 3: %d" % (square_of_sums - sum_of_squares)

if __name__ == '__main__':
	solve()

File: js/solutions/Problem6.js

exports.solve = function () {
	var util = require("../util"),
		nums = util.range(1, 101),
		squareOfSums = Math.pow(util.sum.apply(null, nums), 2),
		sumOfSquares = util.sum.apply(null, nums.map(function (i) {
			return Math.pow(i, 2);
		}));
	
	return Math.abs(sumOfSquares - squareOfSums);
}